### PR TITLE
[FRONT-1422][FRONT-1440] Removed unused code and removed redux

### PR DESCRIPTION
**Pull request by pmichalowski**
It's kinda impossible to review but you may look at a few things here:
- DataUnion secrets list & editor - I did some changes here
- useDataUnioneSecrets - hook's implementation was replaced with a Context
- removed GlobalInfoWatcher & Web3Poller as in my opinion, they were not doing anything

### CODE REVIEW

File: app/src/marketplace/modules/dataUnion/dataUnionSecrectsContext.tsx
Comment (tumppi) [inline app/src/marketplace/modules/dataUnion/dataUnionSecrectsContext.tsx:1]: Remove extra `c` in the filename dataUnionSecre`c`tsContext.tsx

**Review by tumppi — APPROVED** [review]
RIP precious lines 😢

File: app/src/marketplace/containers/ProjectEditing/DataUnionSecrets/SecretEditor.tsx [hunk 1]
```diff
@@ -65,29 +60,15 @@ export const SecretEditor: FunctionComponent<Props> = ({ secret, chainId, dataUn
                     <Button
                         disabled={name == null || name.length === 0}
                         onClick={async () => {
-                            if (name == null || name.length === 0) {
+                            if (name == null || name.length === 0 || typeof name === 'undefined') {
```
Comment (mondoreale) [inline app/src/marketplace/containers/ProjectEditing/DataUnionSecrets/SecretEditor.tsx:63]:
```js
undefined == null // true
```

File: app/src/marketplace/containers/ProjectEditing/DataUnionSecrets/SecretEditor.tsx [hunk 2]
```diff
@@ -65,29 +60,15 @@ export const SecretEditor: FunctionComponent<Props> = ({ secret, chainId, dataUn
...
-                                        dataUnionId,
-                                        name,
-                                    })
-                                }
-
                                 if (onResolve != null) {
-                                    onResolve()
+                                    onResolve(name)
                                 }
+                                return
```
Comment (mondoreale) [inline app/src/marketplace/containers/ProjectEditing/DataUnionSecrets/SecretEditor.tsx:71]: Unnecessary `return`.

**Review by mondoreale — APPROVED** [review]
17k lines of code gone… I love what I'm seeing and it's nicely executed. o7

See my notes. Not much there.

Cheers!

**Pull request merged**